.github/workflows/build-container.yaml

```
@@ -236,4 +245,64 @@ jobs:
+    strategy:
+      matrix:
+        include:
+          - arch: amd64
+            os: ubuntu-24.04
+          - arch: arm64
+            os: ubuntu-24.04-arm
+            
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Login to GHCR
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)